cards/src/main/resources/cards/custom/group2/set_633/hero_power_effuse.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Looks like the filename hasn't changed but the card ID in the hero has, so please check that

Reply:
This has been resolved